create-pages/intl.js

```diff
@@ -0,0 +1,44 @@
+const langConfig = require('../lang.config.json')
+const defaultLang = langConfig.defaultLang
+const langMapKeys = Object.keys(langConfig.languages)
+const dev404Page = '/dev-404-page'
+
+const createIntlPages = ({ page, actions }) => {
```

Comment:
This createIntlPages will create `/en/` and default `/` for English pages. We have decided to omit langPrefix`/en/`.